src/index.ts

```diff
@@ -256,6 +260,16 @@ export async function createServer() {
     }
   })
 
+  server.route({
```

Comment:
I've added this endpoint to have a somewhat real-world-like way for me to push hospital notifications in. The handler itself needs some love if and when we integrate it DHIS2 as some FHIR entities have hard-coded values and so on. Notice also that there's no death handler now.